<p>Java Application Framework class naming convention.</p>
<h2>Noncompliant Code Example</h2>
<pre>
	public class MyRestCapability extends BusinessService {
	
	}
</pre>
<h2>Compliant Solution</h2>
<pre>
	public class MyRestCapability extends RestCapability {
		
	}
</pre>

