<p>Transfer Object variable naming convention.</p>
<h2>Noncompliant Code Example</h2>
<pre>
	public class TONameVerify {

	boolean booleanVar;
	byte byteVar;
	short shortVar;
	long longVarTO;
	
}

</pre>
<h2>Compliant Solution</h2>
<pre>
	public class TONameVerify {

	boolean booleanVar;
	byte byteVar;
	short shortVar;
	long longVar;
	Employee empTO;
}

</pre>

